hardhat.config.ts

```diff
@@ -1,73 +1,81 @@
 import 'dotenv/config';
 
 import { HardhatUserConfig } from 'hardhat/config';
-import { NetworkUserConfig } from "hardhat/types";
+import { NetworkUserConfig } from 'hardhat/types';
 import '@nomicfoundation/hardhat-toolbox';
 import '@openzeppelin/hardhat-upgrades';
 import 'hardhat-tracer';
 import '@nomiclabs/hardhat-solhint';
 import 'hardhat-contract-sizer';
 import 'hardhat-storage-layout-changes';
+import 'hardhat-abi-exporter';
 import './tasks/deploy';
 import './tasks/update-module';
 import './tasks/upgrade';
 
 type SSVNetworkConfig = NetworkUserConfig & {
   ssvToken: string;
-}
-
+};
 
 const config: HardhatUserConfig = {
   // Your type-safe config goes here
   mocha: {
-    timeout: 40000000000000000
+    timeout: 40000000000000000,
   },
   solidity: {
     compilers: [
       {
-        version: "0.8.4",
+        version: '0.8.4',
       },
       {
         version: '0.8.18',
         settings: {
           optimizer: {
             enabled: true,
-            runs: 10000
-          }
-        }
-      }
+            runs: 10000,
+          },
+        },
+      },
     ],
   },
   networks: {
     ganache: {
       chainId: 1337,
-      url: "http://127.0.0.1:8585",
-      ssvToken: process.env.SSVTOKEN_ADDRESS // if empty, deploy SSV mock token
+      url: 'http://127.0.0.1:8585',
+      ssvToken: process.env.SSVTOKEN_ADDRESS, // if empty, deploy SSV mock token
     } as SSVNetworkConfig,
     hardhat: {
-      allowUnlimitedContractSize: true
-    }
+      allowUnlimitedContractSize: true,
+    },
   },
   etherscan: {
     // Your API key for Etherscan
     // Obtain one at https://etherscan.io/
     apiKey: process.env.ETHERSCAN_KEY,
     customChains: [
       {
-        network: "holesky",
+        network: 'holesky',
         chainId: 17000,
         urls: {
-          apiURL: "https://api-holesky.etherscan.io/api",
-          browserURL: "https://holesky.etherscan.io"
-        }
-      }
-    ]
+          apiURL: 'https://api-holesky.etherscan.io/api',
+          browserURL: 'https://holesky.etherscan.io',
+        },
+      },
+    ],
   },
   gasReporter: {
     enabled: true,
     currency: 'USD',
-    gasPrice: 0.3
-  }
+    gasPrice: 0.3,
+  },
+  abiExporter: {
+    path: './abis',
+    runOnCompile: true,
+    clear: true,
+    flat: true,
+    spacing: 2,
+    pretty: false,
+  },
```

Comment:
Users will only interact with our main contracts SSVNetwork and SSVNetworkViews. So only these ABIs are needed. I think using the `only` property here you can got it.